_scripts/instruction-widget/install.js

```diff
@@ -183,6 +188,7 @@ module.exports = function(context) {
       context.package = "certbot-nginx";
     }
     context.dns_plugins = true;
+    context.dns_package_prefix = "certbot-dns";
```

Comment:
This should be `python3-certbot-dns`.

(It appears the aliases without the `python-` prefix weren't added for our DNS plugins. I talked to our Fedora maintainer who said they'd add them, but they said that we should tell people to use `python3-*` in the meantime.)